.github/workflows/ci.yml

```
@@ -30,7 +30,7 @@
           sudo apt-get update
           sudo apt-get install -y build-essential
       - name: Build project
-        run: make all
+        run: make all -j $(nproc)
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:13: Quote this to prevent word splitting

(shellcheck)